### ISSUE TITLE
Feature Request - List of profiles

### ISSUE THREAD

**Issue by raarellano** [amplicity/AlfredWorkflow-NewBraveWindow#1]
Hi, I wanted to pitch a feature request to allow the workflow to show a list of profiles to choose which profile to open the new window in. 

This is the [Alfred Chrome Workflow](https://github.com/ShogunPanda/alfred-chrome) I use has this same feature for Chrome.

Thanks in advance if this is possible!

**Comment by amplicity** [comment]
@raarellano thanks for bringing this to my attention. Will take a look, it's most likely possible. 

If you figure out how, and would like to submit a PR, i'll take a look!

**Comment by denys-chaikovskyi** [comment]
That would be really awesome to open a new window with a specific profile right from the Alfred 👍

**Comment by amplicity** [comment]
Yes, agreed. Unfortunately I haven't had the time to work on this, but encourage anyone else to create a PR if they figure it out.
